hs_communities/static/css/bootstrap.3.0.2.css

```diff
@@ -0,0 +1,7098 @@
+/*!
```

Comment:
We are already serving this file here:
https://github.com/hydroshare/hydroshare/blob/develop/theme/static/css/bootstrap.3.0.2.min.css

Reply:
Yes, but the collectstatic command reconciles that. The Django pattern is to include all the needed files in the Django app, so if they change in another app, the dependencies are not broken.

Reply:
We kind of break away from that with the theme app, which is arguably bad/legacy practice. I'm just going for the maximally compatible and serviceable approach here -- which also happens to be best practice.

Reply:
Wouldn't this add unnecessary loading time if each app serves the same files?

Reply:
That's not my understanding of how collectstatic works. Have you read up on it/worked with it a lot?

Reply:
I haven't, but it looks like we are at least repeating files in the code base which seems unintuitive. Perhaps we should have a common folder with files meant to be used by apps we develop.

Reply:
I'll explain this if we can get on a call. It's how Django is supposed to work and helps avoid entagling dependencies for JS libraries; allows for library updates (such as jQuery, VueJS) to be more granular/incremental and avoid having to refactor multiple Django apps during an update.

Reply:
I'm going to mark this as resolved. Hopefully, it makes sense after explaining: imagine you have 10+ Django Apps, like we do. If you want to upgrade bootstrap, jQuery you need to change the code in all the apps and know which one is using which version. If each Django App controls its own static resources, then you can upgrade one at a time for smaller, more coherent PRs. Throughout these upgrades, consolidation for deployment is handled automatically, with the use of `collectstatic`